plugins/lookup/onepassword.py

```diff
@@ -38,6 +38,10 @@
         type: str
       subdomain:
         description: The 1Password subdomain to authenticate against.
+      account_id:
+        description: The account ID to target.
+        type: str
+        version_added: 7.5.0
```

Comment:
This option should also be added to `onepassword_raw.py` in the same directory, since it shares the code of this plugin.